### PR TITLE
Re-export Feature to be compatible with newer versions of Features.

### LINKED ISSUE #12: Feature needs to be updated to match features-7.x-2.6
With `features-7.x-2.6`, this Feature installs in an overridden state due to `foreign key` issues and overridden VBO actions.

It needs to be re-exported.

### PR DESCRIPTION
**Pull request by jbickar**
Fixes #12

### CODE REVIEW

File: stanford_video.info [hunk 1]
```diff
@@ -13,6 +13,7 @@ dependencies[] = image
 dependencies[] = jw_player
 dependencies[] = link
 dependencies[] = nodeformcols
+dependencies[] = stanford_video
```
Comment (sherakama) [inline stanford_video.info]: stanford_video depends on .... stanford_video?

File: stanford_video.views_default.inc [hunk 2]
```diff
@@ -152,13 +152,6 @@ function stanford_video_views_default_views() {
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_settings']['force_single'] = 0;
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_settings']['entity_load_capacity'] = '10';
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_operations'] = array(
-    'action::node_assign_owner_action' => array(
```
Comment (sherakama) [inline stanford_video.views_default.inc:155]: Please bring these VBO actions back.

File: stanford_video.info (hunk 1 above)
Comment (jbickar) [inline stanford_video.info]: I was elected to lead, not to read.

File: stanford_video.views_default.inc (hunk 2 above)
Comment (jbickar) [inline stanford_video.views_default.inc:155]: OK, let me figure out why it installs as overridden and can't be reverted, then.

Comment (sherakama) [inline stanford_video.views_default.inc:155]: There is a reference to stanford slides in the vbo actions and I am not sure why.

Comment (sherakama) [inline stanford_video.views_default.inc:155]: If need be, pull the vbo view out and put it in its own stanford_video_administration module

Comment (jbickar) [inline stanford_video.views_default.inc:155]: Actually, I should install this on a clean Drupal build with only the required dependencies and re-export the Feature.

Comment (jbickar) [inline stanford_video.views_default.inc:155]: Those actions are not available in the View when installed on a clean Drupal. So there's either an unmet dependency, or the syntax for these actions has changed. Yarr.

**Comment by jbickar** [comment]
Available operations now are:
- Change author
- Delete
- Modify
- Publish
- Unpublish

Some of the other ones didn't make sense.

**Comment by sherakama** [comment]
Fine by me.

**Pull request merged**